### PR TITLE
removed Visual Studio Code from pre-requirements

### PR DESCRIPTION
**Pull request by aremo-ms**
Removed Visual Studio Code from pre-requirements

### CODE REVIEW

**Comment by derisen** [comment]
@aremo-ms should we remove this command as well, if they will be using Visual Studio only?

https://github.com/Azure-Samples/ms-identity-dotnetcore-ca-auth-context-app#step-2-install-project-dependencies

**Comment by aremo-ms** [comment]
> @aremo-ms should we remove this command as well, if they will be using Visual Studio only?
> 
> https://github.com/Azure-Samples/ms-identity-dotnetcore-ca-auth-context-app#step-2-install-project-dependencies

I have no problem to remove because it will be ran by 'dotnet run' command.
@kalyankrishna1 - your opinion?

File: README.md [hunk 1]
```diff
@@ -104,14 +97,14 @@ There is one project in this sample. To register it, you can:
 
 </details>
 
-### Choose the Azure AD tenant where you want to create your applications
+### Step 2 (Alternative Manual setup): Choose the Azure AD tenant where you want to create your applications
```
Comment (derisen) [inline README.md:101]: @aremo-ms do we need this heading (`Step 2 (Alternative Manual setup)`)? the section above (previous step 2) reads as:

> There is one project in this sample. To register it, you can:
> 
> follow the steps below for manually register your apps
> or use PowerShell scripts that:
> automatically creates the Azure AD applications and related objects (passwords, permissions, dependencies) for you.
> modify the projects' configuration files.
> Expand this section if you want to use this automation:

**Review by derisen — APPROVED** [review]
@aremo-ms LGTM! one comment

**Pull request merged**